tests/modules/nf-core/limma/differential/nextflow.config

```diff
@@ -0,0 +1,21 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+    
+    withName: 'test_limma_differential:LIMMA_DIFFERENTIAL' {
+        ext.args = { [
+            "--sample_id_col name",
+            "--contrast_variable $meta.variable",
```

Comment:
Are this values depending on the sample? Just wondering if the standard way is to pass them through meta.

Reply:
So, this is a copy of the way deseq2/differential does it, which is the product of a few cycles of review, so I _think_ it's okay.

In #differentialabundance the contrasts are loaded from a file and passed to the differential module(s) one-by-one. Originally I was having the templates read the contrast meta directly via `meta.variable` etc, and I was then sagely told to do it this way to avoid making assumptions about `meta` content. 

I think value channels would be a much better way- but you know the discussion around that ;-).

Reply:
fair enough, look good then :) thanks for the clarification